src/dmd/root/outbuffer.d

```diff
@@ -57,22 +94,48 @@ struct OutBuffer
         return p;
     }
 
+    /**
+    Releases all resources associated with `this` and resets it as an empty
+    memory buffer. The config variables `notlinehead`, `doindent` etc. are
+    not changed.
+    */
     extern (C++) void destroy() pure nothrow @trusted
     {
-        debug (stomp) memset(data.ptr, 0xFF, data.length);
-        mem.xfree(extractData());
+        if (fileMapping && fileMapping.active)
+        {
+            fileMapping.close();
+            data = null;
+            offset = 0;
+        }
+        else
+        {
+            debug (stomp) memset(data.ptr, 0xFF, data.length);
+            mem.xfree(extractData());
+        }
     }
 
+    /**
+    Reserves `nbytes` bytes of additional memory (or file space) in advance.
```

Comment:
Use Ddoc style comments.

Reply:
that IS a ddoc comment. the /++ ... +/ model is of course superior but still

Reply:
> that IS a ddoc comment. the /++ ... +/ model is of course superior but still

Missing `Params:` and `Returns:` ;-)

Reply:
Added `Params`, it returns `void` so no need for `Returns`.

Reply:
Right. Void returns do not need a `Returns:`.